CHANGELOG.md

```diff
@@ -3,6 +3,20 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v0.2 - [04/05/2022]
```

Comment:
This file is for when we actually do a pipeline release, not for step by step changes.

Reply:
This has now been removed in commit 7e0a660.